DOCS-3693: Update module and namespace renaming (#4418)
viamrobotics/docs: docs/operate/get-started/other-hardware/manage-modules.md

## Rename a module

You can rename a module that your organization owns through the Viam web interface.
To rename a module:

1. Navigate to your module page at `app.viam.com/module/<namespace>/<module-name>`.
1. Click the **...** menu in the top right corner of the module page.
1. Select **Rename** from the dropdown menu.
1. Enter the new module name in the modal that appears.
1. Click **Rename** to confirm the change.

> When you rename a module, the old module name is reserved for backwards compatibility cannot be reused.

Reviewer: ```suggestion
When you rename a module, Viam reserves the old module name for backwards compatibility and you cannot reuse it.
```

## Rename a module

You can rename a module that you own through the Viam web interface.
To rename a module:

1. Navigate to your module page at `app.viam.com/module/<namespace>/<module-name>`.
1. Click the **...** menu in the top right corner of the module page.
1. Select **Rename** from the dropdown menu.
1. Enter the new module name in the modal that appears.
1. Click **Rename** to confirm the change.

Notes: 

- The old module name is permanently reserved for backwards compatibility and cannot be reused.
- Existing machine configurations will continue to work without any changes required.
- When adding the module to new machines, the machine builder will use the new module name.
- You can only rename modules that your organization owns.

> ## Rename a model

Reviewer: ```suggestion
{{% hiddencontent %}}

## Rename a model
```
